OPHD/UI/PopulationPanel.h

```diff
@@ -23,6 +24,7 @@ class PopulationPanel : public Control
 	PopulationPanel();
 
 	void population(Population* pop);
+	void populationPool(PopulationPool* popPool);
```

Comment:
Largely unrelated to this PR, though I noticed it during review and wanted to note this construct as an area for potential future work. I opened Issue https://github.com/OutpostUniverse/OPHD/issues/1442 to elaborate on it.